app/models/service_template_provision_task.rb

```diff
@@ -111,6 +111,9 @@ def queue_post_provision
 
   def do_post_provision
     update_request_status
+
+    _log.info("Queuing post provision of #{self.class.name} id: #{id}, tracking_label_id: #{tracking_label_id}, zone: #{zone}")
```

Comment:
Some log messages would have `tracking_label_id` prefix added. How about this one? 
Just don't want `tracking_label_id` showing up twice in one log message.